### PR TITLE
Add base for bitnami kafka from Helm chart in pubsub repo

### PR DESCRIPTION
**Pull request by sbuliarca**
- Add almost vanilla bitnami/kafka base
- Back to basics
- Example kminion base
- Deploy bitnami exporter for kafka-shared
- Enable jmx metrics for kafka-shared
- Changed the namespace for the cluster

### CODE REVIEW

File: kminion/values.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /metrics
+    prometheus.io/port: "8080"
+# TODO: This is to generate an example config.yaml in a configMap. It might be
+# a better idea to actually have that configMap defined downstream.
+kminion:
+  config:
+    kafka:
+      brokers:
+        - kafka-shared-0.kafka-shared-headless.sys-pubsub.svc.cluster.local:9093
```
[the remark this answers is not in the document]
Comment (ffilippopoulos) [reply kminion/values.yaml:13]: I tried that as an alternative to the jmx exporter, this is why it's there. But I do not think there is much value on it, so I wouldn't copy it over here

[the remark this answers is not in the document]
Comment (sbuliarca) [reply kminion/values.yaml:13]: I didn't notice it. Seems richer than the default exporters that ships with bitnami.

[the remark this answers is not in the document]
Comment (sbuliarca) [reply kminion/values.yaml:13]: > I tried that as an alternative to the jmx exporter, this is why it's there. But I do not think there is much value on it, so I wouldn't copy it over here

I'll drop it from the PR then. We can come back to it later if we need to

**Review by ffilippopoulos — APPROVED** [review]

File: bitnami/Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+BITNAMI_KAFKA_RELEASE=21.4.2
```
Comment (rcrowe) [inline bitnami/Makefile:1]: Any reason to not move to 22 yet?

Comment (sbuliarca) [reply bitnami/Makefile:1]: I'll include the upgrade

Comment (sbuliarca) [reply bitnami/Makefile]: Upgrade included => only the helm version annotation was updated.

**Review by george-angel — APPROVED** [review]

**Pull request merged**